java/vector/src/main/codegen/templates/AbstractPromotableFieldWriter.java

```diff
@@ -314,11 +331,6 @@ public MapWriter map(String name, boolean keysSorted) {
     return getWriter(MinorType.STRUCT).${lowerName}(name);
   }
 
-  @Override
-  public ${capName}Writer ${lowerName}() {
-    return getWriter(MinorType.LIST).${lowerName}();
-  }
-
```

Comment:
Removing this doesn't change any public APIs?

Reply:
If someone used to use `AbstractPromotableFieldWriter` then the answer is yes, but if someone used `PromotableWriter` then it won't be an issue.

Reply:
Would you prefer not making this change, but override it in the `PromotableViewWriter`?

Reply:
To be safe I would rather override rather than move functions in these core APIs

Reply:
Let's do that